content/en/docs/multi-center/modules/icu/d_items.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "d_items"
+linktitle: "d_items"
+weight: 2
+date: 2023-09-18
+description: >
+  Dimension table describing itemid. Defines concepts recorded in the events table in the ICU module.
+---
+
+**Links to:** 
+
+* *chartevents* on `itemid`
+* *procedureevents* on `itemid`
+
+# Table columns
+
+Name | Postgres data type
+---- | ----
+itemid | INTEGER
+label | VARCHAR(200)
+abbreviation | VARCHAR(100)
+linksto | VARCHAR(50)
+category | VARCHAR(100)
+unitname | VARCHAR(100)
+param\_type | VARCHAR(30)
+lownormalvalue | FLOAT
+highnormalvalue | FLOAT
+
+# Detailed Description
+
+The *d_items* table defines `itemid`, which represents measurements in the database. Measurements of the same type (e.g. heart rate) will have the same `itemid` (e.g. 220045). Values in the `itemid` column are unique to each row. All `itemid`s will have a value > 220000.
+
+To faciliate future multi-center initiatives, we will release mappings within charted events (including vital signs) to standard terminologies (LOINC) and mappings for procedure events to standard terminologies (SNOMED).
```

Comment:
I would not promise future work and instead link to the mimic-code repo where we have already created some MIMIC mappings

Reply:
added the link.